app/components/crate-sidebar/install-instructions.hbs

```diff
@@ -0,0 +1,76 @@
+{{#if @binNames}}
+  {{#if (is-clipboard-supported)}}
+    <CopyButton
+      @copyText={{this.cargoInstallCommand}}
+      title="Copy command to clipboard"
+      local-class="copy-button"
+    >
+      <span local-class="selectable">{{this.cargoInstallCommand}}</span>
+      {{svg-jar "copy" aria-hidden="true" local-class="copy-icon"}}
+    </CopyButton>
+  {{else}}
+    <code local-class="copy-fallback">
+      {{this.cargoInstallCommand}}
+    </code>
+  {{/if}}
+
+  <p local-class="copy-help">
+    Running the above command will globally install the following binaries:
```

Comment:
I hate saying this, but should we make this singular/plural based on the number of binaries? (I guess if this looks like a list enough maybe it doesn't matter.)

Reply:
I've moved the sentence into the condition blocks and special cased it for the 1- and 2-item cases